packages/myst-frontmatter/src/frontmatter/validators.ts

```diff
@@ -317,9 +333,10 @@ export function validateAffiliation(input: any, opts: ValidationOptions) {
   if (defined(value.institution)) {
     output.institution = validateString(value.institution, incrementOptions('institution', opts));
   }
-  if (!output.name && !output.institution) {
-    validationWarning('affiliation should include name or institution', opts);
-  }
+  // It is possible to have an ID only at this point
+  // if (!output.name && !output.institution) {
+  //   validationWarning('affiliation should include name or institution', opts);
+  // }
```

Comment:
I removed this warning -- maybe there is a better place to put it, or perhaps if there is a single object with just ID, we skip it (that is better than removing it. I will push.).